tools/unitctl/unit-client-rs/src/unitd_instance.rs

```diff
@@ -107,6 +119,24 @@ impl UnitdInstance {
 
         fn map_process_to_unitd_instance(process: &UnitdProcess) -> UnitdInstance {
             match unitd_path_from_process(process) {
+                Ok(_) if process.container.is_some() => {
+                    let mut err = vec![];
+                    // double check that it is running
+                    let running = process.container.as_ref().unwrap().container_is_running();
```

Comment:
Would this unwrap be better as an expect?

Reply:
I dont think so, since we already ruled out that process.container is None (see line 122)

Reply:
Whatever message gets put in there I think would be superfluous and never seen.